docs/cvl/methods.md

```diff
@@ -178,6 +190,66 @@ shown in the web report can indicate whether a summary was applied.
 
 ```
 
+(catch-unresolved-calls-entry)=
+### Catch unresolved-calls entry
+Example:
+```cvl
+methods {
+   function _._ external => DISPATCH [
+      C.foo(uint),
+      _.bar(address), // Will resolve to all available functions with the signature "bar(address)", specifically Other.bar(address)
+      C._ // Will resolve to all functions in C, specifically C.foo(uint) and C.baz(bool)
+   ] default NONDET;
+}
+```
+
+The catch unresolved-calls entry is a special type of summary declaration that instructs the Prover to replace calls to unresolved external function calls with a specific kind of summary, dispatch list.
+By default, the Prover will use an AUTO summary for unresolved function calls, but that may produce spurious counter examples. 
+The catch unresolved-calls entry lets the user refine the summary used for unresolved function calls.
+
+```{note}
+Only one catch unresolved-calls entry is allowed per a specification file. When importing a specification with a catch unresolved-calls entry it will be included as part of the current specification, and cannot be overridden.
+```
+
+Catch unresolved-calls entry can only be summarized with a dispatch list summary (and a dispatch list summary is only applicable for a catch unresolved-calls entry). 
```

Comment:
```suggestion
A catch unresolved-calls entry can only be summarized with a dispatch list summary (and a dispatch list summary is only applicable for a catch unresolved-calls entry). 
```
Also - maybe hyphenate `dispatch-list`?

Reply:
I don't think it is a good idea to hyphenate as sometimes I will address `dispatch list` without saying summary (and those cases won't need hyphens).